app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt

```diff
@@ -65,7 +65,8 @@ class ProfileFragment : Fragment() {
                 }
                 .subscribe({ user ->
                     Timber.d("Response Success")
-                    rootView.name.text = "${user.firstName} ${user.lastName}"
+                    if (!(user.firstName.isNullOrEmpty() && user.lastName.isNullOrEmpty()))
+                        rootView.name.text = "${user.firstName} ${user.lastName}"
```

Comment:
A user may have first name and no last name, and vice versa

Do this

"${firstname.nullToEmpty()} ${lastname.nullToEmpty()"

Reply:
for null strings, this would mean "true true". Not sure this will work

Reply:
nullToEmpty != isNullOrEmpty